src/Routing/PurlRouteProvider.php

```diff
@@ -163,7 +163,9 @@ public function getRouteCollectionForRequest(Request $request) {
         'query' => $query_parameters,
         'routes' => $routes,
       ];
-      $this->cache->set($cid, $cache_value, CacheBackendInterface::CACHE_PERMANENT, ['route_match']);
+      if ($routes->all() || $routes->getResources()) {
```

Comment:
Can't we just use one of them? either `all` or `getResources`?

Reply:
It is working fine with getResources(). I wanted to test it on dev, so added both. Removing all().